main.go

```diff
@@ -1,20 +1,21 @@
 package main
 
 import (
-	"fmt"
 	"os"
 
+	"github.com/Sirupsen/logrus"
 	"github.com/urfave/cli"
+	"github.com/joho/godotenv"
 )
 
-var build string // build number set at compile-time
+var version string // build number set at compile-time
```

Comment:
Looks like we need update `.drone.yml`, to set version https://github.com/drone-plugins/drone-git/blob/master/.drone.yml#L12